client/src/app/pages/identities/identities.tsx

```diff
@@ -47,6 +46,8 @@ import {
 } from "@app/queries/identities";
 import { useFetchApplications } from "@app/queries/applications";
 import { NotificationsContext } from "@app/shared/notifications-context";
+import { IdentityForm } from "./components/identity-form";
+import { validateXML } from "./components/identity-form/validateXML";
```

Comment:
Not needed

Reply:
I am sorry - we may still need this. Need to check with Jeff/Ramon here about support for mvn identities for mta 7. I Know we will need it for v0.2.0